src/main/resources/scripts/parse_console_for_cbt.py

```diff
@@ -28,66 +27,85 @@ def parse(self, console_log):
 
         for line in console_log:
             line = line.strip()
-            
+                        
             if line.startswith("Processing options file"):
-                if started:
```

Comment:
It's difficult to tell what the change was.
Could you give me a brief explanation of what was changed here?

Reply:
Had to change the parsing of the build log for "monitored mode, built from a regression script" which was different from any of the other build logs we've seen; therefore, I had to restructure the processing of that file accordingly.